backend/z_index.js

```diff
@@ -41,6 +41,42 @@ app.get('/meetingHistory', async (req,res) => {
   const meetingHist = await getMeetingHistory();
   res.json(meetingHist);
 })
+
+app.get('/MeetingBarData', async (req, res) => {
+  var type = req.query.type;
```

Comment:
Generally, we dont want to name our backend endpoints after what the frontend is going to use it for since things may change and we may want to use this data for things other than the meeting bar.

I would name the endpoints based on what data they provide

1. Looks like MeetingBar Data will be used to get a specific meeting type -> meeting/{type} where {type} is a query parameter

2. DonutData looks like one where we specify both type and meeting number -> meeting/{type}/{number}

More about query parameters and query strings in express: https://stackabuse.com/get-query-strings-and-parameters-in-express-js/